gravatar-ui/src/main/java/com/gravatar/ui/components/atomic/Avatar.kt

```diff
@@ -27,15 +31,23 @@ import com.gravatar.ui.skeletonEffect
  * @param size The size of the avatar
  * @param modifier Composable modifier
  * @param avatarQueryOptions Options to customize the avatar query
+ * @param forceRefresh While this is true, we'll force the refresh of the avatar in every recomposition
  */
 @Composable
 public fun Avatar(
     profile: Profile,
     size: Dp,
     modifier: Modifier = Modifier,
     avatarQueryOptions: AvatarQueryOptions? = null,
+    forceRefresh: Boolean = false,
 ) {
     val sizePx = with(LocalDensity.current) { size.roundToPx() }
+    var cacheBuster by rememberSaveable { mutableStateOf<String?>(null) }
+
+    if (forceRefresh) {
+        cacheBuster = System.currentTimeMillis().toString()
+    }
+
```

Comment:
When `forceRefresh` is true the `cacheBuster` will be updated on each recomposition. What's the advantage of using `rememberSaveable` for it?

Reply:
At somepoint `forceRefresh` can be switched to `false` and, for example, rotate the device. The idea is to still use the previous cacheBuster.